### PR TITLE
chore(clerk-js): Fix broken Organization methods

### PR DESCRIPTION
**Pull request by panteliselef**
## Description

This PR fixes 2 methods that seems to be flawed.

https://github.com/clerkinc/javascript/blob/214f658cf79983b8b44ae3228b0500d88e215345/packages/clerk-js/src/core/resources/Organization.ts#L325

https://github.com/clerkinc/javascript/blob/214f658cf79983b8b44ae3228b0500d88e215345/packages/clerk-js/src/core/clerk.ts#L1086

To provide you with more context, these 2 methods rely on endpoints that resolve with paginated data and the iterate through them and filter the array based on an organizatinoID. This concept is flawed, as there is great chance that the organization id you are looking for, might not be part of the first page of the results.

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerkinc/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Review by anagstef — APPROVED** [review]
💯

File: packages/clerk-js/src/core/resources/Organization.ts [hunk 1]
```diff
@@ -342,18 +342,16 @@ export class Organization extends BaseResource implements OrganizationResource {
...
-      {
+
+    const json = (
+      await BaseResource._fetch<OrganizationJSON>({
+        path: `/organizations/${this.id}`,
         method: 'GET',
-        path: `/me/organization_memberships`,
         rotatingTokenNonce,
-      },
-      { forceUpdateClient: true },
```
Comment (chanioxaris) [inline packages/clerk-js/src/core/resources/Organization.ts:351]: Don't we need to include this parameter as well?

**Review by chanioxaris — APPROVED** [review]

File: packages/clerk-js/src/core/resources/Organization.ts (hunk 1 above)
Comment (panteliselef) [reply packages/clerk-js/src/core/resources/Organization.ts:351]: brought it back

File: .changeset/unlucky-carrots-exist.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/clerk-react': patch
+'@clerk/types': patch
+---
+
+Fix methods in clerk-js that consumede paginated endpoints in order to retrieve single resources.
```
Comment (dimkl) [inline .changeset/unlucky-carrots-exist.md:7]: 🙃 Should we replace `consumede` with `consume` ?

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**